Feat/plt 447 readme (#431)
tatumio/tatum-js: README.md

This repository is a monorepo with multiple packages for each blockchain.

### Blockchain subpackage

You can select one or more blockchain packages that you want to use in your project and install them separately.

```console
npm install @tatumio/eth
```

Currently supported blockchains
| Syntax | Description |
| ----------- | ----------- |
| Algo | @tatumio/algo |
| Bitcoin cash | @tatumio/bch |
| Bitcoin | @tatumio/btc |
| Binance smart chain | @tatumio/bsc |
| Celo | @tatumio/celo |
| Dogecoin | @tatumio/doge |
| Ethereum | @tatumio/eth |
| Flow | @tatumio/flow |
| KuCoin | @tatumio/kcs |
| Klaytn | @tatumio/klaytn |
| Litecoin | @tatumio/ltc |
| Polygon | @tatumio/polygon |
| Solana | @tatumio/solana |
| Tron | @tatumio/tron |
| Stellar | @tatumio/xlm |
| XRP | @tatumio/xrp |

### Full package

> There is possibility to install full package with all supported blockchain although due to installation of all dependencies and size usage is not recommended for frontend apps.

Reviewer: reword: It is possible to install the full package with all supported blockchain modules, although this is not recommended for browser environments due to the size of the dependencies.

seems more clear to me like that?

Author: ```suggestion
It is possible to install the full package with all supported blockchain modules, although this is not recommended for browser environments due to the size of the dependencies.
```
